src/EditorFeatures/Core.Wpf/Suggestions/FixAll/FixAllGetFixesService.cs

```diff
@@ -42,7 +42,7 @@ public async Task<Solution> GetFixAllChangedSolutionAsync(IFixAllContext fixAllC
             }
 
             fixAllContext.CancellationToken.ThrowIfCancellationRequested();
-            return await codeAction.GetChangedSolutionInternalAsync(fixAllContext.Solution, cancellationToken: fixAllContext.CancellationToken).ConfigureAwait(false);
+            return await codeAction.GetChangedSolutionInternalAsync(fixAllContext.Solution, fixAllContext.ProgressTracker, cancellationToken: fixAllContext.CancellationToken).ConfigureAwait(false);
```

Comment:
many codepaths had portions that *could* take progress, but were passed 'null' progress sinks.  Those APIs were updated to have progress objects passed in, which generally bubbled up all the way to points like this where we could then connect the external progress UI to the internal progress reporting apis.  This is a good thing and represents more places now where the user will see useful progress updates.